curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md

```diff
@@ -6,7 +6,7 @@ challengeType: 2
 
 ## Description
 <section id='description'>
-As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
+For the following challenges, you will be working with a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
```

Comment:
@adam-weiler Quick question: Should it be `For the following challenge` or `For the following challenges`?

Reply:
This starter project is used for lessons 12, 13, and 14 of the unit:
-Understand BCrypt Hashes
-Hash and Compare Passwords Asynchronously
-Hash and Compare Passwords Synchronously

Lessons 1-11 of the "Information Security with HelmetJS" unit use a different starter project. That's why I thought it is important to point out the difference, so students realize to fork the new starter.

Reply:
OK, just confirming.